integrationTests/DonationPersistenceTest.php

```diff
@@ -32,20 +32,6 @@ public function testItSavesADonationToDB(): void
         $this->assertRowsSimilar($this->donationRow(), $donationRow);
     }
 
-
```

Comment:
the test above checks that it can save and then load from the DB. IMHO it's not especially useful to just check loading in isolation, and now that campaign_id is non-nullable this is more awkward to set up as we have to insert the charity and campaign into db before we can manually insert a donation.